google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudType.java

```diff
@@ -157,8 +156,10 @@ public ManagedVmDeploymentConfiguration createDefaultConfiguration(
         @NotNull DeploymentSource source,
         @NotNull RemoteServer<ManagedVmServerConfiguration> server) {
       return new ManagedVmDeploymentRunConfigurationEditor(project, source,
-          new CloudSdkAppEngineHelper(new File(server.getConfiguration().getCloudSdkExecutablePath()),
-              server.getConfiguration().getCloudProjectName()));
+          new CloudSdkAppEngineHelper(
+              new File(server.getConfiguration().getCloudSdkExecutablePath()),
+              server.getConfiguration().getCloudProjectName(),
+              server.getConfiguration().getGoogleUserName()));
```

Comment:
I think I found an edge case here: if the user already has a cloud configured but is not logged in any more for whatever reason, then this line throws an NPE when trying to get back into the deployment editor.